contracts/test/MapleTreasury.t.sol

```diff
@@ -2,22 +2,30 @@
 pragma solidity 0.6.11;
 pragma experimental ABIEncoderV2;
 
-import { IERC20 } from "../../../../../lib/openzeppelin-contracts/contracts/token/ERC20/IERC20.sol";
+import { IERC20 }        from "../../modules/openzeppelin-contracts/contracts/token/ERC20/IERC20.sol";
+import { Util }          from "../../modules/util/contracts/Util.sol";
+import { IMapleGlobals } from "../../modules/globals/contracts/interfaces/IMapleGlobals.sol";
+import { DSTest }        from "../../modules/ds-test/src/test.sol";
 
-import { Util } from "../../../../libraries/util/contracts/Util.sol";
+import { Governor }    from "./accounts/Governor.sol";
+import { MapleTreasury } from "../MapleTreasury.sol";
```

Comment:
Since these are different "things" in the hierarchy of imports.
```suggestion
import { MapleTreasury } from "../MapleTreasury.sol";

import { Governor } from "./accounts/Governor.sol";
```